src/storage/sr-module.entity.ts

```diff
@@ -0,0 +1,81 @@
+import { Entity, EntityRepositoryType, PrimaryKey, PrimaryKeyType, Property } from '@mikro-orm/core';
+import { StakingModule } from '../staking-router-modules/interfaces/staking-module.interface';
+import { SRModuleRepository } from './sr-module.repository';
+
+@Entity({ customRepository: () => SRModuleRepository })
+export class SrModuleEntity {
+  [EntityRepositoryType]?: SRModuleRepository;
+  [PrimaryKeyType]?: [number, string];
```

Comment:
To honour DRY principle you can write `[PrimaryKeyType]?: [SrModuleEntity['id'], SrModuleEntity['stakingModuleAddress']];`